session/bootstrap.go

```diff
@@ -865,6 +871,12 @@ func upgradeToVer35(s Session) {
 	mustExecute(s, sql)
 }
 
+func upgradeToVer36(s Session) {
+	doReentrantDDL(s, "ALTER TABLE mysql.user ADD COLUMN `Shutdown_priv` ENUM('N','Y') DEFAULT 'N'", infoschema.ErrColumnExists)
+	// A root user will have those privileges after upgrading.
+	mustExecute(s, "UPDATE HIGH_PRIORITY mysql.user SET Shutdown_priv='Y' where User = 'root'")
```

Comment:
I think grant `SHUTDOWN` to users which have `SUPER` privilege is better. For those delete `root` and create another admin user, simply  grant `SHUTDOWN` to root may cause bootstrap fail.

Reply:
`UPDATE HIGH_PRIORITY mysql.user SET Shutdown_priv='Y' where Super_priv = 'Y'`